source/nodejs/adaptivecards-designer/README.md

```diff
@@ -227,5 +232,13 @@ For advanced configuration of the designer use the following APIs.
 			]
 		}
 	);
+
+	/* Modify the Element toolbox (BEFORE designer attached) */ 
```

Comment:
Curious - what does "after/before designer attached" imply?

Reply:
It means before the designer instance's `attachTo` method is called.